modules/auxiliary/admin/kerberos/get_ticket.rb

```diff
@@ -67,6 +68,12 @@ def initialize(info = {})
         )
       ]
     )
+
+    register_advanced_options(
+      [
+        kerberos_cache_mode_option(default: 'write-only')
```

Comment:
Should this option be deregistered? It doesn't look like it does anything any more since from what I can tell we define the authenticator and the cache mode explicitly for each operation now.